contrib/runners/inquirer/runner.yaml

```diff
@@ -0,0 +1,37 @@
+- description: A runner for Inquiries
+  enabled: true
+  name: inquirer
+  runner_module: inquirer
+  runner_parameters:
+    schema:
+      default:
+        title: response_data
+        type: object
+        properties:
+          continue:
+           type: boolean
+           description: Would you like to continue the workflow?
+        required: ["continue"]
+      required: true
+      description: A JSON schema that will be used to validate the response data
+      type: object
+    tag:
+      default: ""
+      required: false
+      description: An arbitrary value for tracking the purpose of the action (useful in rules)
+      type: string
+    roles:
+      default: []
+      required: false
+      description: A list of roles that are permitted to respond to the action (if nothing provided, all are permitted) - REQUIRES ENTERPRISE FEATURES 
+      type: array
+    users:
+      default: []
+      required: false
+      description: A list of usernames that are permitted to respond to the action (if nothing provided, all are permitted)
+      type: array
+    ttl:
+      default: 1440
+      required: true
+      description: Time (in minutes) that an unacknowledged Inquiry is cleaned up
```

Comment:
Maybe re-word it a bit - something along the lines of "Number of minutes to wait before timing out the inquiry if no response / ack is received by then".

Reply:
Fixed in fee4118b7